gemx.gemspec

```diff
@@ -1,10 +1,7 @@
-lib = File.expand_path('../lib', __FILE__)
-$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'gemx/version'
```

Comment:
how do you feel about `require_relative './lib/gemx/version' instead?

Reply:
I just put it in a plain text file